### PR TITLE
Various New Features cf #8

### PR DESCRIPTION
**Pull request by DerGuteWolf**
based on steampunk branch!

### CODE REVIEW

**Comment by DerGuteWolf** [comment]
I don't know where the conflict came from but should now be resolved

**Comment by sbcgua** [comment]
sorry for some delay, really busy these days. Will try to review during the weekend

**Comment by DerGuteWolf** [comment]
Hi @sbcgua ,

did you find time already to take a look at this?

Regards,
  Wolfgang

File: src/zif_mustache.intf.abap [hunk 1]
```diff
@@ -1,7 +1,7 @@
 interface zif_mustache
   public .
 
-  constants version type string value '2.0.0'. " Package version
+  constants version type string value '2.1.0'. " Package version
```
Comment (sbcgua) [inline src/zif_mustache.intf.abap]: Please don't adjust the versions, let it to me

**Review by sbcgua — CHANGES_REQUESTED** [review]
Sorry for so late response, really much work recently. Also the PR is huge, it would be much faster if it were step-by-step.
So there are some small adjustments I would do, but probably let's merge it and see how it goes. All this looks like good and useful features
But please remove the version change, I'd like to maintain versioning myself.

File: src/zif_mustache.intf.abap (hunk 1 above)
Comment (DerGuteWolf) [reply src/zif_mustache.intf.abap]: ok, sorry, I reverted the version change.

Comment (DerGuteWolf) [reply src/zif_mustache.intf.abap]: sorry for the delay, I was awfully busy the last weeks...

**Comment by sbcgua** [comment]
Yeap, merging. Thank you for the contribution!

P.S. "I was awfully busy the last weeks" - same for me, too much work ... I definitely need to do something with it ... :)

**Pull request merged**